src/_pytest/unittest.py

```diff
@@ -187,29 +191,45 @@ def addSuccess(self, testcase):
     def stopTest(self, testcase):
         pass
 
-    def _handle_skip(self):
-        # implements the skipping machinery (see #2137)
-        # analog to pythons Lib/unittest/case.py:run
+    def runtest(self):
         testMethod = getattr(self._testcase, self._testcase._testMethodName)
-        if getattr(self._testcase.__class__, "__unittest_skip__", False) or getattr(
-            testMethod, "__unittest_skip__", False
-        ):
-            # If the class or method was skipped.
-            skip_why = getattr(
-                self._testcase.__class__, "__unittest_skip_why__", ""
-            ) or getattr(testMethod, "__unittest_skip_why__", "")
-            self._testcase._addSkip(self, self._testcase, skip_why)
-            return True
-        return False
 
-    def runtest(self):
-        if self.config.pluginmanager.get_plugin("pdbinvoke") is None:
+        class _GetOutOf_testPartExecutor(KeyboardInterrupt):
+            """Helper exception to get out of unittests's testPartExecutor."""
+
+        unittest = sys.modules.get("unittest")
```

Comment:
At this point `unittest` has definitely been imported, no need to guard against that. 👍